lib/journaled/audit_log.rb

```diff
@@ -61,6 +63,14 @@ def classic_exclude!(name)
       rescue NameError
         nil
       end
+
+      def independent_class_in_7_1?(name)
+        return false if Gem::Version.new(Rails.version) < Gem::Version.new('7.1')
+
+        # These classes do not inherit from ActiveRecord::Base in 7.1
+        # so calling `skip_audit_log` on them will raise.
+        %w(ActiveRecord::InternalMetadata ActiveRecord::SchemaMigration).include?(name)
```

Comment:
I am assuming the original reason for skipping these was that we didn't want them to have the callbacks defined in this module, but that they would have them because we `include` the module in the `on_load` hook for AR (i.e. in 7.1, since these classes don't inherit from Base, then they don't get the methods after all)

Reply:
Would it make sense to exclude these from `DEFAULT_EXCLUDED_CLASSES` at class load time if we're on 7.1 or greater?

Reply:
(And, yeah, that was the goal. If they don't inherit, then they don't need to be included.)

Reply:
Alternatively, we could check `.respond_to?(:skip_audit_log)`, but I would start with the more explicit list.

Reply:
Makes total sense; I updated the const's assignment and removed this method! I felt like inlining the array for both branches of the if-expr wasn't that bad/verbose (and decided to drop it by a line to avoid having a huge indent).